setup.py

```diff
@@ -172,7 +181,8 @@
     "s3": [
         "fsspec[s3]",
```

Comment:
```suggestion
        _deps["fsspec"],
```
to have compatible versions, since `fsspec[s3]` is installing no pinned `s3fs` version. https://github.com/intake/filesystem_spec/blob/bb7c23305ce76cb22cd10305f87a49009f363a11/setup.py#L50